lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs

```diff
@@ -350,9 +349,12 @@ runIO db@DBLayer{..} = fmap Resp . go
     go = \case
         CleanDB -> do
             Right . Unit <$> cleanDB db
-        -- FIXME Create a wallet with already some tx history
-        CreateWallet wid wal meta -> catchWalletAlreadyExists (const (NewWallet (unMockWid wid))) $
+        CreateWallet wid wal meta ->
+            catchWalletAlreadyExists (const (NewWallet (unMockWid wid))) $
             mapExceptT atomically $ initializeWallet (widPK wid) wal meta mempty
+        CreateWalletWithTxs wid wal meta txs ->
+            catchWalletAlreadyExists (const (NewWallet (unMockWid wid))) $
+            mapExceptT atomically $ initializeWallet (widPK wid) wal meta txs
```

Comment:
*[suggestion]* 

This is a more general case of `CreateWallet`. What about simply fixing the generator for `CreateWallet` instead of creating a new action? Certainly, the generator will sometime yield wallet creation with no txs (and we can even tag it to convince ourselves). So, no need for two separate commands here :)

Reply:
yes, just extended CreateWallet and resigned from CreateWalletWithTxs